spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraCatalog.scala

```diff
@@ -53,7 +37,7 @@ private[cassandra] class CassandraCatalog(cc: CassandraSQLContext) extends Catal
 
   override def tableExists(tableIdentifier: Seq[String]): Boolean = {
     val (cluster, database, table) = getClusterDBTableNames(tableIdentifier)
-    val schema = schemas.get(cluster)
+    val schema = cc.getCassandraSchema(cluster)
     val tabDef = for (ksDef <- schema.keyspaceByName.get(database); tabDef <- ksDef.tableByName.get(table)) yield tabDef
     tabDef.isDefined
   }
```

Comment:
`CassandraCatalog` has some unimplemented methods. Is it correct? Can you add comments explaining why they can be left unimplemented? E.g. `getTables` method or `refreshTable` doesn't look right.